lib/cocoapods/installer/pod_source_installer.rb

```diff
@@ -191,6 +199,14 @@ def local?
         sandbox.local?(root_spec.name)
       end
 
+      # @return [Boolean] whether the pod uses an external source (e.g. :podspec) in the
+      #         resolution process to retrieve its podspec.
+      #
+      def external?
+        @dependencies ||= podfile.dependencies.select(&:external?).map(&:name)
```

Comment:
will this ever actually be different from `released?`? Should the ivar be named `external_dependencies`?

Reply:
oh interesting. I can check and see. I can rename the ivar yes.